pkgs/applications/editors/rstudio/default.nix

```diff
@@ -170,35 +212,95 @@ stdenv.mkDerivation rec {
       done
     done
 
-    ln -s ${quarto} dependencies/quarto
+    ln -s ${quartoWrapper} dependencies/quarto
 
     # version in dependencies/common/install-mathjax
     ln -s ${mathJaxSrc} dependencies/mathjax-27
 
-    # version in CMakeGlobals.txt (PANDOC_VERSION)
-    mkdir -p dependencies/pandoc/2.18
-    ln -s ${lib.getBin pandoc}/bin/* dependencies/pandoc/2.18
+    # node used by cmake
+    # version in CMakeGlobals.txt (RSTUDIO_NODE_VERSION)
+    mkdir -p dependencies/common/node
+    ln -s ${nodejs} dependencies/common/node/18.18.2
 
+    # node used at runtime
     # version in CMakeGlobals.txt (RSTUDIO_INSTALLED_NODE_VERSION)
     mkdir -p dependencies/common/node
```

Comment:
```suggestion
```
this line is unnecessary, I have removed it in the next PR. (don't really want to cause more rebuilds with this PR)